spec/requests/preferences/preference_spec.rb

```diff
@@ -81,4 +81,68 @@
       end
     end
   end
+
+  describe 'GET edit' do
+    context 'when logged in' do
+      let!(:user) { create(:user) }
+      let!(:preference) { create(:preference, user_id: user.id) }
+      let!(:id) do
+        preference.id
+      end
+
+      before { sign_in user }
+
+      it 'have http status 200' do
+        expect(get(edit_preference_path(preference.id))).to eq(200)
+      end
```

Comment:
Maybe you could add another `it` block to test that some attribute of the preference (for example `preference.name`) is correctly returned.

Reply:
Done! :slightly_smiling_face:

Reply:
👏  Nice work!